ui-core/src/styles/select.css

```diff
@@ -1,15 +1,19 @@
-.custom-select {
+.osrd-ui-custom-select {
   border-radius: 4px;
   border: 1px solid rgba(148, 145, 142, 1);
   box-shadow:
     0px 1px 1px rgba(0, 0, 0, 0.2),
     0px 2px 4px rgba(0, 0, 0, 0.1);
   height: 2.5rem;
-  padding: 0.438rem 0.813rem 0.562rem 0.813rem;
+  padding: 0.438rem 2rem 0.563rem 0.813rem;
```

Comment:
If I remove everything for just a `padding-left: 0.813rem;`, the position stays unchanged. Do we need all this configuration (I agree it matches the mockup, but it matches it also without it)